fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/InvalidSerializationFormatExceptionMapper.java

```diff
@@ -42,13 +42,10 @@
 
     @Override
     public Response toResponse(final InvalidSerializationFormatException e) {
-        if (LOGGER.isDebugEnabled()) {
-            LOGGER.debug("InvalidSerializationFormatException intercepted by InvalidSerializationFormatExceptionMapper"
-                    + (e.getMessage() != null ? ": " + e.getMessage() : "."), e);
-        } else {
-            LOGGER.info("InvalidSerializationFormatException intercepted by InvalidSerializationFormatExceptionMapper"
+
+        LOGGER.error("InvalidSerializationFormatException intercepted by InvalidSerializationFormatExceptionMapper"
```

Comment:
Instead of String concatenation in this message with `+`, please use the template syntax of `{}`.